### ISSUE TITLE
Enable deferred button for archived assignments

### ISSUE THREAD

**Issue by leonieherma1** [hypha-dao/dho-web-client#1489]
Topic: Change deferred amount for expired assignment in order to be able to claim remaining periods
https://dao.hypha.earth/hypha/proposals/30792

Expected: being able to change from 80% to 100% deferred
Experienced: cannot find a button to do that

<img width="1063" alt="image" src="https://user-images.githubusercontent.com/75991832/185338776-07be8b45-757c-4687-8789-49705a5ff4a6.png">

blocked by backend story: https://github.com/hypha-dao/dao-contracts/issues/298

**Comment by leonieherma1** [comment]
@evgenibir I tried to change deferred amount to 80% like 5 times on this assignment: https://dao-dev.hypha.earth/hypha/proposals/30377

but it is not working:
<img width="502" alt="image" src="https://user-images.githubusercontent.com/75991832/189150509-1d9f8b64-38ee-4e66-a29c-a92d5a2aa071.png">

**Comment by evgenibir** [comment]
> @evgenibir I tried to change deferred amount to 80% like 5 times on this assignment: https://dao-dev.hypha.earth/hypha/proposals/30377
> 
> but it is not working: <img alt="image" width="502" src="https://user-images.githubusercontent.com/75991832/189150509-1d9f8b64-38ee-4e66-a29c-a92d5a2aa071.png">

Hello @leonieherma1! I have tested it, and saw that problem associated with backend. I get this error in console "app.1f88907e.js:203 Error: Error: assertion failure with message: Cannot change deferred percentage onarchived assignments". I think we can close this issue, and open new issue for backend developers

**Comment by leonieherma1** [comment]
@Gerard097 or @Alex-IS can you take a look at this error? 

@evgenibir can you try again? I sometimes get this error and then try again and then it works

**Comment by evgenibir** [comment]
> /hypha/proposals/30377

@leonieherma1, I tried to do this 4-5 times and I always get an error, this is a backend error, I think it is necessary to remove the check on whether this assignment is active or not

**Comment by leonieherma1** [comment]
ok @evgenibir can we remove the check? or is that a problem?

**Comment by leonieherma1** [comment]
https://user-images.githubusercontent.com/75991832/204950283-e0013394-fc73-4576-b4f2-5d1d38b8c78c.mov

@evgenibir the "deferred button" is enabled, but if I change the deferred to 100% it is not changing (it stays at 76%)
